### ISSUE TITLE
Blurry Images

### ISSUE THREAD

**Issue by LuisBurdallo** [QL-Win/QuickLook#1037]
It seems the app doesn't work properly when using Windows Scaling feature which makes images look blurry. I guess it scales up small images. Images larger than your resolution wouldn't be affected in the same way. Can we keep the scaling untouched? I tried overriding the DPI scaling in the exe properties but it didn't help at all.

**Comment by vivadavid** [comment]
I've noticed the same behaviour too.

**Comment by xupefei** [comment]
You can CTRL+Wheel to zoom down to 100%.

**Comment by vivadavid** [comment]
> You can CTRL+Wheel to zoom down to 100%.

Thank you: I didn't know that! I wonder if it would be possible to add "CTRL+" and "CTRL-" as shorcuts.

In any case, it would be nice if users could set a particular default behaviour as otherwise you need to use the zoom for each individual image. In my personal case scenario, I'd leave images at 100% at all times, except when images don't fit the size of the screen.

**Comment by LuisBurdallo** [comment]
Yeah that's ok for one picture but when you have to check 25 because that's part of your job then it is not a good solution. I am currently using Seer which is amazing and it doesn't have that issue. I wanted to use Quicklook because some explorers do not support Seer.

**Comment by xupefei** [comment]
Actually this behavior is easy to change by adding a check in [this file](https://github.com/QL-Win/QuickLook/blob/master/QuickLook.Plugin/QuickLook.Plugin.ImageViewer/ImagePanel.xaml.cs) to see if it's the first time loading an image. Anyone want to take a stab?
